resources/content/schema/project/project-auth.json

```diff
@@ -77,6 +77,7 @@
         "serviceBinding" : "cru",
         "volumeTemplate" : "cr",
         "volumeActivateInput" : "cr",
-        "end" : ""
+        "end" : "",
```

Comment:
"end" should always be at the end.  The purpose is so that people don't forget that the last element doesn't have `,`.